src/pudl/transform/classes.py

```diff
@@ -674,7 +713,7 @@ class TableTransformParams(TransformParams):
 
     @classmethod
     def from_dict(cls, params: dict[str, Any]) -> "TableTransformParams":
-        """Construct TableTransformParams from a dictionary of constant parameters."""
+        """Construct ``TableTransformParams`` from a dictionary of constants."""
```

Comment:
what are constants?

Reply:
Added more details / pointer to the params subpackage.